app/src/main/java/ca/ualberta/CMPUT3012019T02/alexandria/controller/SearchController.java

```diff
@@ -24,6 +35,10 @@
     private Index index;
     private Gson gson;
     private static SearchController instance;
+    private final String GOOGLE_BOOK_URL = "https://www.googleapis.com/books/v1/volumes?&maxResults=1&projection=lite&q=";
+    private final String apiKey = "AIzaSyA0Km9mozkCqNX7N0Sy4V_Uk1OIvxUvRn4";
```

Comment:
This is a public repo. I would not commit api keys because anyone can use them. Ty shared `secrets.xml`. Probably better to add to that

Reply:
will do. i'll move the algolia one as well